README.md

```diff
@@ -170,15 +171,15 @@ As mentioned in the example above, you may optionally specify a file type along
 type of file extension, just make sure you have the appropriate loaders enabled.
 
 For example, `js` will always work by default. However, `css` will only work if the `css-loader` is enabled. If you have
-the `less-loader` enabled, you can do something like this:
+the `less-loader` or `sass-loader` enabled, you can do something like this:
 
 ```twig
 <section>
-    {% webpack inline less %}
+    {% webpack inline less|sass %}
```

Comment:
I was trying to say that you can use either less or sass in inline mode.
I will update the readme with the code style